### PR TITLE
subscription to reports 

### PR DESCRIPTION
**Pull request by horlathunbhosun**

### CODE REVIEW

**Comment by horlathunbhosun** [comment]
image

File: app/Http/Controllers/ExpenseController.php [hunk 1]
```diff
@@ -20,4 +22,19 @@ public function ministry()
         $collection['summary'] = Payment::where('description', '!=', '')->paginate(20);
         return view('pages.expense.ministry')->with('collection', $collection);
     }
+
+
+
+    public function store(Request $request){
```
Comment (goodhands) [inline app/Http/Controllers/ExpenseController.php:28]: The store method in `ExpenseController` should be for storing Expenses and not Subscription. You have the SubscriptionController for that

File: database/migrations/2020_07_13_170758_create_subscriptions_table.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('subscriptions', function (Blueprint $table) {
+            $table->id();
+            $table->string('email') ;
```
Comment (goodhands) [inline database/migrations/2020_07_13_170758_create_subscriptions_table.php:19]: Please add a column of `name` which should be nullable

File: resources/views/pages/expense/index.blade.php [hunk 3]
```diff
@@ -32,7 +33,24 @@
...
+            <form method="POST" action="{{ url('expense/report')}}">
+              @csrf
+              <input type="text" class="form-control" name="email" placeholder="Enter Your Email" style="width: 50%;">
+              <br>
+              <select name="subscription_type" id="" class="form-control" style="width: 50%;">
+                  <option value="">Select The Report type</option>
+                  <option value="ministry">Ministry</option>
+                  <option value="contract">Contract</option>
+                  <option value="daily&expense">Daily Expense</option>
+                  <option value="payment">Payments W/0 description</option>
```
Comment (goodhands) [inline resources/views/pages/expense/index.blade.php]: Please change this to something more descriptive. `Payments without subscription` is fine or something else the user can relate with

**Review by goodhands — CHANGES_REQUESTED** [review]
Make the changes and update your PR

File: database/migrations/2020_07_13_170758_create_subscriptions_table.php (hunk 2 above)
Comment (horlathunbhosun) [reply database/migrations/2020_07_13_170758_create_subscriptions_table.php:19]: ok will do that

File: resources/views/pages/expense/index.blade.php (hunk 3 above)
Comment (horlathunbhosun) [reply resources/views/pages/expense/index.blade.php]: will do that thanks

Comment (horlathunbhosun) [reply resources/views/pages/expense/index.blade.php]: @goodhands  fixed all what you said

**Comment by horlathunbhosun** [comment]
@goodhands  fixed all changes

**Comment by goodhands** [comment]
Thank you @horlathunbhosun

**Pull request merged**